include/gdf/cffi/types.h

```diff
@@ -44,6 +44,7 @@ typedef enum {
     GDF_JOIN_DTYPE_MISMATCH,          /**< Datatype mismatch between corresponding columns in  left/right tables in the Join function */   
     GDF_JOIN_TOO_MANY_COLUMNS,        /**< Too many columns were passed in for the requested join operation*/       
     GDF_GROUPBY_TOO_MANY_COLUMNS,
+    GDF_IO_ERROR,
```

Comment:
I don't think it is good practice to insert into the middle of an enum.  Also, can you please provide a docstring. Thanks!

Reply:
Ok! Thanks for the feedback.